libs/sdk-backend-mockingbird/src/compositeBackend/index.ts

```diff
@@ -48,7 +48,7 @@ export function compositeBackend(...components: CompositeBackendPart[]): IAnalyt
     const config = primaryBackend.config;
 
     const backend: IAnalyticalBackend = {
-        capabilities: {},
+        capabilities: primaryBackend.capabilities,
```

Comment:
Do we want to do some intersection or something here? 🤔  but I guess that would depend on particular capabilities, right?

Reply:
that would be tricky / impossible even with the way how we have it today. theoretically you can create composite from multiple different types of backends. the intersection would be 0. however the capabilities would actually be depending on which component does the processing land.

the only way would be to move the capabilities to workspace as well and get them from there (and they would be coming from the actual component in that case and be correct)

since this is testing infra, i will document the behavior and limitations of the compositeBackend.

Reply:
Ok, that makes sense.